dali/python/nvidia/dali/backend.py

```diff
@@ -22,7 +22,6 @@
 # for importing the DALI c++ extensions, we can do it here
 
 default_plugins = [
-    'libpython_function_plugin.so'
```

Comment:
Does it stop being loaded by default or is there an other way of using it that I'm missing?

Reply:
This was not needed as it doesn't work like that. In ops.py we import nvidia.dali.python_function_plugin and that is all we needed. Loading so here didn't have any effect as it only registers the operator but it doesn't initialize it as a Python module.